config/webpack/environment.js

```diff
@@ -25,4 +25,11 @@ environment.loaders.append('style', {
   ]
 })
 
+const nullLoader = {
+  test: /\.css$/,
+  include: stylesheet => stylesheet.indexOf('@patternfly/react-styles/css/') > -1,
+  use: ['null-loader']
+}
+environment.loaders.insert('null', nullLoader, {before: 'style'})
```

Comment:
Instead of `insert` before 'styles' you could call `append` before appending the styles loader.

Also the styles loader is declared in 1 line, I suggest we do the same with `nullLoader` so that the file looks consistent.

Reply:
@josemigallas done!